beetsplug/lyrics.py

```diff
@@ -77,6 +78,11 @@ class HTMLParseError(Exception):
 
 # Utilities.
 
+def unichar(i):
+    try:
+        return six.unichr(i)
+    except ValueError:
+        return struct.pack('i', i).decode('utf-32')
```

Comment:
I had such an exception occuring once in the `unichar` code with one of the page returned by the Google backend.